plugins/faustwp/tests/integration/replacement/test-replacement-callbacks.php

```diff
@@ -35,8 +35,12 @@ public function test_preview_post_link_filter() {
 		$this->assertSame( 1000, has_action( 'preview_post_link', 'WPE\FaustWP\Replacement\post_preview_link' ) );
 	}
 
```

Comment:
Is it possible at this point to include e2e tests as well? It would be great if we had them for reference.